plugins/pipx/plugin.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "pipx",
+  "version": "0.1.0",
+  "description": "Installs Pipx and makes it available",
+  "packages": ["pipx@1.4"],
```

Comment:
Unless you don't care about Python version, it's better to use version specified form, e.g.:
`python312Packages.pipx@latest`

Reply:
well we *kind of* don't care about python version, since this is *only* used for installing helper tooling (poetry). The virtualenv itself is still a different python version specified by the app (that said, I think that the 3.11 which this is giving us might be causing netskope issues, so we might want to try 3.9)